lib/tessel/deploy.js

```diff
@@ -780,13 +760,32 @@ actions.tarBundle = function(opts) {
     });
   });
 
+  logs.info('Building project.');
+
+
   // Both the --slim and --full paths will use a copy of the
   // project to bundle. This allows us to be destructive
   // with the files, but without directly tampering with the
   // project files themselves.
   var tempBundleDir = fsTemp.mkdirSync();
+  var common = ['node_modules', 'package.json', opts.resolvedEntryPoint];
+  var assets = fs.readdirSync(target).filter(entry => (common.indexOf(entry) === -1));
 
-  logs.info('Building project.');
+  if (assets.length) {
+    logs.warn(
+      tags.stripIndent `
+        The following items were found in your project directory, but not deployed.
+
+          ${assets.join('\n')}
+
+        If you need to deploy them to your Tessel 2, you must create a .tesselinclude
```

Comment:
If the project wasn't started with `t2 init`, they will have to create the file. We could also say "type `t2 init`", but what if they _did_ and purposely deleted the `.tesselinclude` file, thinking that they didn't need to use it? It might seem strange to say "run `t2 init`" to someone that already did that.